.goreleaser.yml

```diff
@@ -0,0 +1,18 @@
+project_name: 'mm-geofeed-verifier'
+archive:
+    wrap_in_directory: true
+    replacements:
+        darwin: Darwin
+        linux: Linux
+        windows: Windows
+        386: i386
+        amd64: x86_64
+    files:
+        - CHANGELOG.md
+        - LICENSE-APACHE
+        - LICENSE-MIT
+        - README.md
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
```

Comment:
I just wanted to double check that you did this deliberately. I think you did and it looks fine, I just noticed that it wasn't present in `mm-network-analyzer`'s GoReleaser config so I figured I'd check.

Reply:
I *think* I initially believed that things should be setup for Travis-CI and that this made that feasible/easier. I'm going to remove it for now.